azure.yaml

```diff
@@ -3,6 +3,11 @@
 name: todo-nodejs-mongo-aks
 metadata:
   template: todo-nodejs-mongo-aks@0.0.1-beta
+hooks:
+  predeploy:
+    interactive: false
+    shell: sh
+    run: chmod +x infra/hooks/predeploy.sh && infra/hooks/predeploy.sh # installs additional components on the cluster
```

Comment:
Is the chmod required? I would expect that it would be added to the git repo already with the execute permissions set.

Reply:
Got you, I will double check it, will remove if it's not necessary.